README.md

```diff
@@ -1,2 +1,3 @@
 # Hello-World
-This is my first Microverse Hello World repository 
+
+This is my first Microverse Hello World repository
```

Comment:
-You did a great job by adding the readme file but kindly the  README must be descriptive - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project. :+1: